extensions/kusto/src/features.ts

```diff
@@ -59,13 +59,19 @@ export class AccountFeature implements StaticFeature {
 			return undefined;
 		}
 
-		const tenant = account.properties.tenants.find((t: { [key: string]: string }) => request.authority.includes(t.id));
 		const unauthorizedMessage = localize('kusto.insufficientlyPrivelagedAzureAccount', "The configured Azure account for {0} does not have sufficient permissions for Azure Key Vault to access a column master key for Always Encrypted.", account.key.accountId);
-		if (!tenant) {
-			window.showErrorMessage(unauthorizedMessage);
-			return undefined;
+		let tenantId: string = '';
+
+		if (request.provider === 'AzureMFA') {
```

Comment:
I wonder if the condition should be switch to something like `if (request.provider !== 'dtsAuth')`?  i.e. is providing a non-empty tenantId going to be more common with various new auth types.